css/style.css

```diff
@@ -0,0 +1,84 @@
+/**
+ * Place your custom styles here.
+ */
+
+ /*GLOBAL OVERRIDES*/
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+@import url('https://fonts.googleapis.com/css?family=Oswald');
+p,li,dt,dd,td,th,details,button,form,a{font-family: Raleway;}
```

Comment:
Need to specify one or more fallbacks: `font-family: Raleway, serif`  (I'm assuming this is a serif font, otherwise use `sans-serif` instead.  You can also put any number of more common fonts in between, e.g. `font-family: Raleway, Times New Roman, serif` (only please don't use TNR!))